lib/google.rb

```diff
@@ -0,0 +1,11 @@
+require "net/http"
```

Comment:
One of the changes that the `actionkit_connector` gem brings to the table is using HTTParty, which is a cleaner interface for `net/http`. I don't think this needs to change, but I'd flag it with a TODO that we'll want to clean it up to only use one library in the future.